sites/mcro.py

```diff
@@ -0,0 +1,45 @@
+import requests
+import json
+import uuid
+import os
+from bs4 import BeautifulSoup
+
+company = 'Mcro'
+base_url = 'https://mcro.tech'
+careers_url = base_url + '/careers/'
+
+response = requests.get(careers_url)
+html_markup = response.text
+
+soup = BeautifulSoup(html_markup, 'html.parser')
+
+jobs = soup.find_all('a', class_='job')
+
+final_jobs = []
+
+for job in jobs:
+    job_link = job['href']
+    job_title = job.find('h4').get_text(strip=True)
+    location = job.find('div', class_='job__description--location').find('h5').get_text(strip=True)
+    
+
+    # Concatenate base_url with job_link
+    full_job_link = base_url + "/" + job_link
+
+    final_jobs.append({
+        'id': str(uuid.uuid4()),
+        'job_link': full_job_link,
+        'job_title': job_title,
+        'location': location,
```

Comment:
Location must contain only city, and key is city not location
```suggestion
        'city':'Mioveni',
```